apps/financial-aid/web-veita/src/components/History/History.tsx

```diff
@@ -48,6 +48,9 @@ const History = ({ className, applicantName }: Props) => {
           </Text>
 
           {data?.applicationEvents.map((item, index) => {
+            if (!getEventType[item.eventType]) {
+              return null
+            }
```

Comment:
Does this happen at any time?

Reply:
This happened if I didn't migrate the backend, so I put this in there just in case, should I take it out?

Reply:
We probably should right?